### PR TITLE
Fixes to local CI script

### PR DESCRIPTION
**Pull request by shwina**

### CODE REVIEW

File: ci/local/build.sh [hunk 1]
```diff
@@ -80,7 +80,7 @@ BUILD_SCRIPT=${REPO_PATH_IN_CONTAINER}/ci/gpu/build.sh
 if [ -f \${PREBUILD_SCRIPT} ]; then
     source \${PREBUILD_SCRIPT}
 fi
-yes | source \${BUILD_SCRIPT}
```
Comment (shwina) [inline ci/local/build.sh:83]: `yes` seemed to hang the script for me for some reason. Do we need it here?

Comment (ajschmidt8) [reply ci/local/build.sh:83]: Hmm. This script predates me. I don't see why we'd need it here though. I think it's fine to remove.

File: ci/local/build.sh [hunk 2]
```diff
@@ -131,7 +131,7 @@ DOCKER_MAJOR=$(docker -v|sed 's/[^[0-9]*\([0-9]*\).*/\1/')
 GPU_OPTS="--gpus device=${NVIDIA_VISIBLE_DEVICES}"
 if [ "$DOCKER_MAJOR" -lt 19 ]
 then
-    GPU_OPTS="--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES='${NVIDIA_VISIBLE_DEVICES}'"
+    GPU_OPTS="--gpus=all -e NVIDIA_VISIBLE_DEVICES='${NVIDIA_VISIBLE_DEVICES}'"
```
Comment (ajschmidt8) [inline ci/local/build.sh:134]: Any reason this needs to be changed? I think the `--runtime=nvidia` syntax is needed to support users with Docker versions less than 19. This syntax is similar to what we recommend in our DockerHub READMEs (see screenshot, [link](https://hub.docker.com/r/rapidsai/rapidsai-core-dev-nightly))


image

Comment (shwina) [reply ci/local/build.sh:134]: Thanks for the context! Does `--runtime-nvidia` work with newer versions at all? IIRC docker wasn't happy with it and it's often a pain point for me (need to look back in my notes for what to change it to).

Comment (ajschmidt8) [reply ci/local/build.sh:134]: I believe it should. I have the latest version of Docker installed on my machine and I use `--runtime=nvidia` regularly

**Comment by vyasr** [comment]
@shwina I retargeted this since I don't think it's critical (it's only for local builds).

**Pull request closed without merging**